lib/coradoc/document/audio.rb

```diff
@@ -8,19 +8,14 @@ def initialize(title, options = {})
         @id = options.fetch(:id, nil)
         @anchor = Inline::Anchor.new(@id) if @id
         @src = options.fetch(:src, '')
-        @options = options.fetch(:options, [])
+        @attributes = options.fetch(:attributes, [])
       end
 
       def to_adoc
         anchor = @anchor.nil? ? "" : "#{@anchor.to_adoc}\n"
         title = ".#{@title}\n" unless @title.empty?
-
-        opts = ""
-        if @options.any?
-          opts = %{options="#{@options.join(',')}"}
-        end
-
-        [anchor, title, "audio::", @src, "[", opts, "]"].join("")
+        attrs = @attributes.empty? ? "\[\]" : @attributes.to_adoc
```

Comment:
Since we have `AttributeList`, you can implement `AttributeList.to_adoc` which returns `[]` when empty. So this line can go away, and the next line uses `attrs.to_adoc` instead of `attrs`.